src/app/core/components/mmsai-account-button/mmsai-account-button.component.scss

```diff
@@ -0,0 +1,47 @@
+:host {
+  position: relative;
+  max-width: 220px;
+  .account-button {
+    min-width: 0;
+    width: 100%;
+    color: inherit;
+    font-family: inherit;
+    font-size: inherit;
+    border: none;
+    display: flex;
+    gap: 12px;
+    align-items: center;
+    justify-content: center;
+    background: var(--primary-900);
+    padding: 0 8px 0 12px;
+    height: 48px;
+    border-radius: 12px;
+    text-decoration: none;
+    cursor: pointer;
+    transition: 0.2s;
+
+    &:active {
+      transform: scale(0.95);
+    }
+
+    .account-button-label {
```

Comment:
Please no nesting. Your component styles are namespaced to the current component so nothing can go wrong. Please remove the nesting as it makes it easier to read and you don't win anything by replicating the DOM hierarchy in your css.